node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
+    'no-loop-func': error,
+    'valid-jsdoc': warn,
+    'for-direction': error,
+    'no-buffer-constructor': error,
+    'no-new-require': error,
+    'no-path-concat': error,
+    'comma-spacing': error,
+    'comma-style': error,
+    'computed-property-spacing': [ error, never ],
+    'consistent-this': [ error, 'self' ],
+    'func-call-spacing': [ error, never ],
+    'key-spacing': error,
+    'new-cap': error,
+    'no-lonely-if': error,
```

Comment:
Don't allow solitary conditionals within another conditional - `else if` should be used instead.
https://eslint.org/docs/rules/no-lonely-if
```js
// Old
if (condition) {
  // ...
} else {
  if (anotherCondition) {
    // ...
  }
}

// New
if (condition) {
  // ...
} else if (anotherCondition) {
  // ...
}
```

Reply:
what if you want to do 

```js
if (condition) {
  // ...
} else {
  if (anotherCondition) {
    // ...
  } else if (conditionThree) {
    // ...
  }
}
```

? 

just asking

Reply:
I don't think that's allowed, as it should be written as
```js
if (condition) {
  // ...
} else if (anotherCondition) {
  // ...
} else if (conditionThree) {
  // ...
}
```

Reply:
it's actually 

```if (condition) {
  // ...
} else if (anotherCondition) {
  // ...
} else if (!anotherCondition && conditionThree) {
  // ...
}```

Reply:
ok